PhDThesisPSnPDF.cls

```diff
@@ -224,6 +224,11 @@ footnotes.}
 \DeclareOption{lineno}{\PHD@linenotrue}
 \DeclareOption{nolineno}{\PHD@linenofalse}
 
+% Declare new Boolean variable to allow separate toggle 
+% to include author name and title in abstract,
+% i.e. Cambridge BGS style
+\newif\ifntabstract\ntabstractfalse
```

Comment:
```suggestion
\newif\ifPHD@abstracttitle\PHD@abstracttitletrue
\DeclareOption{abstracttitle}PHD@abstracttitletrue}
\DeclareOption{noabstractitle}{\PHD@abstracttitlefalse}
```

Reply:
We should have abstract title on by default as per the university guidelines, so let's have it turned to true.